### PR TITLE
Add footer in REACT #351 fixed

### LINKED ISSUE #351: REACT: Add footer to the REACT version of the project

### PR DESCRIPTION
**Pull request by neeagwl**
#351

### CODE REVIEW

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Two Minor changes I would Recommend 
1.For this import reacteasyemoji which was commented 
use  {emoji("Made with ❤️ in India for the students of the world.")} 
2. Use Black Colour for the footer don't follow the main project's gradient as of now.

File: .eslintcache [hunk 1]
```diff
@@ -1 +1,3 @@
...
...ingCount":0},{"filePath":"38","messages":"39","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"40","messages":"41","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Users\\agarw\\Desktop\\Web Development\\swoc\\doc2pen\\src\\index.js",[],"C:\\Users\\agarw\\Desktop\\Web Development\\swoc\\doc2pen\\src\\reportWebVitals.js",[],"C:\\Users\\agarw\\Desktop\\Web Development\\swoc\\doc2pen\\src\\App.js",[],"C:\\Users\\agarw\\Desktop\\Web Development\\swoc\\doc2pen\\src\\components\\MainContainer\\MainContainer.js",[],"C:\\Users\\agarw\\Desktop\\Web Development\\swoc\\doc2pen\\src\\components\\Footer\\Footer.js",[],"C:\\Users\\agarw\\Desktop\\Web Development\\swoc\\doc2pen\\src\\components\\Header\\Header.js",[],"C:\\Users\\agarw\\Desktop\\Web Development\\swoc\\doc2pen\\src\\containers\\editContext.js",[],"C:\\Users\\agarw\\Desktop\\Web Development\\swoc\\doc2pen\\src\\components\\OutputComponent\\Output.js",[],"C:\\Users\\agarw\\Desktop\\Web Development\\swoc\\doc2pen\\src\\components\\Settings\\Settings.js",[],"C:\\Users\\agarw\\Desktop\\Web Development\\swoc\\doc2pen\\src\\components\\Dropdown\\Dropdown.js",[]]
```
Comment (smaranjitghose) [inline .eslintcache:1]: Remove this please

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
> Two Minor changes I would Recommend
> 1.For this import reacteasyemoji which was commented
> use {emoji("Made with ❤️ in India for the students of the world.")}
> 2. Use Black Colour for the footer don't follow the main project's gradient as of now.
> 3. Bonus tip use proper names in the className so that it will be easy for other's to understand it.

++. Code Readability is must!

**Comment by neeagwl** [comment]
I tried to make the requested changes. Please review it.

**Comment by smaranjitghose** [comment]
Nice work!

**Pull request merged**